applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -19,6 +19,9 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(sw_codec_select, CONFIG_SW_CODEC_SELECT_LOG_LEVEL);
 
+/* Used for debugging, inserts 0 instead of packet loss concealment */
+#define SW_CODEC_OVERRIDE_PLC false
```

Comment:
The PR claims that the KConfig option is unused, but is then replace with a similar non-Kconfig-configurable value. That, to me, implies that it is not unused.

Reply:
Agree. Let me reword the PR.